src/app/pages/BuySovPage/components/BuyForm/index.tsx

```diff
@@ -115,18 +115,25 @@ export function BuyForm() {
               onChange={value => setAmount(value)}
               placeholder="0.0000"
               rightElement={<AssetRenderer asset={Asset.RBTC} />}
+              dataAttribute="buySovPage-enterAmount-field"
             />
             <Slippage>
-              <AvailableBalance asset={Asset.RBTC} />
+              <AvailableBalance
+                asset={Asset.RBTC}
+                dataAttribute="buySovPage-availableBalance-field"
```

Comment:
is "buySovPage" really necessary? since the test is running on a specific the page i think there is no need to indicate the route name for each component.

Reply:
well, that's a question for @JoeStorm99

Reply:
The page name is not completely needed on this page but it's needed on other pages where we have the same elements multiple times, so we should add it here to keep it consistent.